CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# Contributing to SLOTHY
+
+Welcome! We are pleased to see you are interested in contributing to SLOTHY.
+
+In the following, we will lay out a few foundational things that are important
+for your contributions.
+
+## Setup
+The setup for the development works just as the one we present in the
+[`README`](https://github.com/slothy-optimizer/slothy/blob/main/README.md).
```

Comment:
Can you change the links to just be relative links, please? 
This can get very confusing if you work in other branches.

Reply:
Thanks!